sw/airborne/modules/loggers/sdlog_chibios/usbStorage.c

```diff
@@ -205,16 +191,15 @@ static const USBDescriptor* getDescriptor(USBDriver* usbp, uint8_t type, uint8_t
 }
 
 /* Handles global events of the USB driver */
-static void usbEvent(USBDriver* usbp, usbevent_t event)
+static void usbEvent(USBDriver *usbp, usbevent_t event)
 {
   (void) usbp;
 
-  switch (event)
-  {
+  switch (event) {
     case USB_EVENT_CONFIGURED:
-      chSysLockFromIsr();
+      chSysLockFromISR();
```

Comment:
Yes, module can be for a specific arch only... e.g. we have lots of vision modules that only compile on the linux arch...
I guess in most cases there are no checks and it will simply fail if you try to use it on another arch.
It would be better to check a define in the c file and throw a nice error if the arch doesn't match...